static/js/reactions.js

```diff
@@ -116,6 +116,7 @@ function filter_emojis() {
     } else {
         reaction_list.removeClass("hide");
     }
+    $(".emoji-popover-emoji-map").perfectScrollbar("update");
     exports.render_reaction_show_list();
```

Comment:
You need to update the scroll position here before calling updating the scroll bar.
```
$(".emoji-popover-emoji-map").scrollTop(0);
$(".emoji-popover-emoji-map").perfectScrollbar("update");
```